memory speed units and plural for memory units

beradrian opened the pull request.

beradrian commented:
Fix for issue #42 and #43.

gentooboontoo commented:
Thank you. Could you split your changes in two commits (memory plurals then memory speeds)?

gentooboontoo commented on src/quantities.js:
```diff
@@ -235,8 +235,12 @@ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
+    "<bit>"  :[["b","bit","bits"], 0.125, "memory", ["<byte>"]],
+
+    /* memory speed */
+    "<Bps>" : [["Bps"], 1.0, "speed", ["<byte>"], ["<second>"]],
```
`Bps` is not a kind of `speed` (It cannot be converted to `m/h` for instance). `speed` is length over time.

beradrian replied:
Then the kind should be `memory speed`.

beradrian commented:
I split the changes into different commits as you requested.

beradrian commented:
How should I add `memory_speed` to `SIGNATURE_VECTOR` and `KINDS`?

beradrian commented:
Here's an example http://plnkr.co/edit/mHjLqo0p5mhv8KPdI785?p=preview

gentooboontoo commented on src/quantities.js:
```diff
@@ -235,8 +235,12 @@ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
     "<rpm>"   :[["rpm"], 2.0*Math.PI / 60.0, "angular_velocity", ["<radian>"], ["<second>"]],
 
     /* memory */
```
Could you replace `memory` by `information` everywhere in the file to keep the library on par with _Ruby-units_?

gentooboontoo commented on src/quantities.js:
```diff
@@ -235,8 +235,12 @@ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
+    "<bit>"  :[["b","bit","bits"], 0.125, "memory", ["<byte>"]],
+    
+    /* memory speed */
+    "<Bps>" : [["Bps"], 1.0, "memory_speed", ["<byte>"], ["<second>"]],
```
Please use `information-rate` instead of `memory_speed`

beradrian commented:
I renamed memory to information and memory_speed to information-rate. What about my question regarding SIGNATURE_VECTOR and KINDS?

gentooboontoo commented:
You don't have to change `SIGNATURE_VECTOR`. Just add `"25599999980": "information_rate"` to `KINDS` (definitively use underscore for kinds, not dash, sorry for the inconvenience).

Please also include tests about bps/Bps and kind.

gentooboontoo commented on src/quantities.js:
```diff
@@ -234,9 +234,13 @@ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
+    "<bit>"  :[["b","bit","bits"], 0.125, "information", ["<byte>"]],
+    
+    /* information rate */
+    "<Bps>" : [["Bps"], 1.0, "information-rate", ["<byte>"], ["<second>"]],
```
dash to be replaced with underscore.

beradrian commented:
I added "information_rate" to KINDS and replaced dash with underscore.
I saw no tests for memory/information. Are you sure you want me to add for information_rate?

gentooboontoo commented:
Merged. I have rebased your commits and refactored tests. Thank you.
